### ISSUE TITLE
about learning_rate 

### ISSUE THREAD

**Issue by venchi012** [BIGBALLON/cifar-10-cnn#13]
Can the callback function pass the learning rate to the optimizer continuously during the iteration? The learning rate is already available when the optimizer is set.

**Comment by BIGBALLON** [comment]
@abing222 check this repo: https://github.com/BIGBALLON/HTD/tree/master/our_Net
this callback can pass the learning rate to the optimizer continuously during the iteration
https://github.com/BIGBALLON/HTD/blob/master/our_Net/our_callbacks.py

**Comment by BIGBALLON** [comment]
![Image](https://github.com/user-attachments/assets/737830de-ba18-4093-a9f3-d7f69713cff1)
